ui/page/root/wallet_list.go

```diff
@@ -259,7 +259,7 @@ func (pg *WalletSelectorPage) walletWrapper(gtx C, item *load.WalletItem) D {
 	}.Layout(gtx,
 		layout.Rigid(func(gtx C) D {
 			return layout.Flex{
-				Axis:      layout.Vertical,
+				Axis:      layout.Horizontal,
 				Alignment: layout.Start,
 			}.Layout(gtx,
```

Comment:
Please reduce the nesting in this `walletWrapper` method. I see that the overall widget is a linear layout that arranges items vertically (top-down).

Not very clear on what follows but from figma, I suppose we can have 3 direct children of this vertical layout, each made into a method to reduce the nesting.
- layoutNameAndBalance (flex with space between, holding name and balance labels)
- layoutUSDBalance(layout.E perhaps, can possibly skip this if USD conversion is disabled)
- layoutSyncStatus (horizontal linear layout perhaps or horizontal flex with equal spacing)

Does this make sense?

Reply:
yes, would help readability